### ISSUE TITLE
External file for variables and environments

### ISSUE THREAD

**Issue by BrianL-STCU** [madskristensen/RestClientVS#35]
**Is your feature request related to a problem? Please describe.**
I'd like to be able to use the REST Client Visual Studio the same way I use in in Visual Studio Code, with variables defined external to the current .http file, and switchable environments.

**Describe the solution you'd like**
Ideally both extensions would use the same settings files (_.vscode/settings.json_ for nonsensitive settings—probably _..\..\.vscode\settings.json_ relative to the .sln file), _%APPDATA%\Code\User\settings.json_ for sensitive secret values (or _%APPDATA%\Code - Insiders\User\settings.json_).

**Describe alternatives you've considered**
A separate file for the Visual Studio REST Client would be acceptable, but hopefully one that can be committed within the git repo, and one for secrets to be excluded. No switchable environments could work, but that would require some redundancy within the .http to cover the various server and secret permutations.

**Additional context**
See Environments & Variables in the [VSCode REST Client](https://marketplace.visualstudio.com/items?itemName=humao.rest-client#environments) docs.

**Comment by SteveIves** [comment]
I agree. I love this extension but having to embed server and port information in each file is kind of a pain for maintainability. It would be great to have the ability to define variables externally. Or, have an "include" capability to include content like variables from a named external file.
